components/seo/social.htm

```diff
@@ -1,31 +1,13 @@
-
-<meta property="og:type" content="{{ include(template_from_string(viewBag.og_type))  ?: 'website' }}" /> 
-<meta property="og:title" content="{{ 
-    include(template_from_string(viewBag.og_title ?: viewBag.meta_title )) 
-    | default(viewBag.title) | seotitle
-}}" />
-<meta property="og:description" content="{{ 
-    include(template_from_string(viewBag.og_description ?: viewBag.meta_description )) 
-    | default(settings.site_description)
-}}" />
-<meta property="og:image" content="{{ 
-    viewBag.og_ref_image ? include(template_from_string(viewBag.og_ref_image)) | media | url  : "" 
-    | default( viewBag.og_image ? ( viewBag.og_image | media | url ) : "" ) 
-    | default( settings.site_image ? ( settings.site_image | media | url ) : "" )
-}}" />
-<meta property="og:url" content="{{ include(template_from_string(viewBag.canonical_url)) | url }}" />
+<meta property="og:site_name" content="settings.og_site_name">
+<meta property="og:type" content="{{ __SELF__.getOgType() }}" />
+<meta property="og:title" content="{{ __SELF__.getOgTitle() }}" />
+<meta property="og:description" content="{{ __SELF__.getOgDescription() }}" />
+<meta property="og:image" content="{{ __SELF__.getOgImage() }}" />
+<meta property="og:url" content="{{ viewBag.canonical_url|url }}" />
 <meta property="og:locale" content="{{ settings.og_locale }}" />
+<meta property="og:video" content="{{ __SELF__.getOgVideo() }}" />
```

Comment:
Why, BTW, we use hardcoded og_locale and do not use the current locale instead?

Reply:
Because, here use with settings arcane seo (default language) and next declared, alternative language.